$OpenBSD: patch-qmake_generators_makefile_cpp,v 1.2 2017/07/17 08:34:37 zhuk Exp $
Don't special case GNU make, BSD make isn't that bad, too.
Index: qmake/generators/makefile.cpp
--- qmake/generators/makefile.cpp.orig
+++ qmake/generators/makefile.cpp
@@ -671,7 +671,7 @@ MakefileGenerator::init()
         }
     }
 
-    if(noIO() || !doDepends() || project->isActiveConfig("GNUmake"))
+    if(noIO() || !doDepends())
         QMakeSourceFileInfo::setDependencyMode(QMakeSourceFileInfo::NonRecursive);
     for(x = 0; x < compilers.count(); ++x)
         initCompiler(compilers.at(x));
